README.md

```diff
@@ -1,4 +1,8 @@
-<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><img align="center" style="margin-bottom:20px;" src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/meshery/meshery-logo-tag-light-text-side.png"  width="70%" /></a><br /><br /></p>
+<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><picture align="center">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-dark-text-side.svg"  width="70%" align="center" style="margin-bottom:20px;">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-light-text-side.svg" width="70%" align="center" style="margin-bottom:20px;">
```

Comment:
Needs modification , light mode doesn't look appropriate .

![image](https://user-images.githubusercontent.com/95926324/193501411-1d5126d6-d22c-4127-9046-4e4474c374f8.png)

Reply:
@legendarykamal It is working fine for me both on light and dark mode. Can you share more details - platform/browser etc
 
![image](https://user-images.githubusercontent.com/61960532/193543708-debedce9-9a82-4811-adca-d047a980eb78.png)

Reply:
@legendarykamal

Reply:
ok @nishantn31  must have been a glitch from my end, rechecked looks fine ,
congrats your first PR got merged 🥳 .